server.js

```diff
@@ -1883,14 +1882,8 @@ if (config.startServer) {
         load.initEstimator('python_callback_waiting', 1);
         callback(null);
       },
-      async () => {
-        if (config.workersExecutionMode === 'container') {
-          await codeCallerDocker.init();
-        }
-      },
-      function (callback) {
-        codeCallers.init();
-        callback(null);
+      async function () {
```

Comment:
Nit for consistency:
```suggestion
      async () => {
```